pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -268,10 +308,35 @@ spec:
               mountPath: /csi
           imagePullPolicy: {{ .ImagePullPolicy }}
         {{ end }}
+        {{ if .CSILogRotation}}
+        - command:
+          - /bin/sh
+          - -c
+          - "\nPERIODICITY={{ .CSILogRotationPeriod }}\nLOG_ROTATE_CEPH_FILE=/etc/logrotate.d/csi\nLOG_MAX_SIZE={{ .CSILogRotationMaxSize }}\n\n#
+            replace default daily with given user input\nsed --in-place \"s/daily/$PERIODICITY/g\" \"$LOG_ROTATE_CEPH_FILE\"\n\n
+            if [ \"$LOG_MAX_SIZE\" != \"0\" ]; then\n\t# adding maxsize $LOG_MAX_SIZE at the 4th line of the logrotate
+            config file with 4 spaces to maintain indentation\n\tsed --in-place \"4i \\
+            \\ \\ \\ maxsize $LOG_MAX_SIZE\" \"$LOG_ROTATE_CEPH_FILE\"\nfi\n\nwhile true;
+            do\n\t# we don't force the logrorate but we let the logrotate binary handle
+            the rotation based on user's input for periodicity and size\n\tlogrotate --verbose
+            \"$LOG_ROTATE_CEPH_FILE\"\n\tsleep 15m\ndone\n"
```

Comment:
Can this be made into a multiline string so it's easier to read, modify, and review?